packages/gatsby-theme-newrelic/README.md

```diff
@@ -656,6 +658,80 @@ const Documentation = () => (
 );
 ```
 
+### `Collapser`
+
+This element is used to reveal or hide content associated with it. Use in
+conjunction with a [`CollapserGroup`](#collapsergroup) when using multiple
+`Collapser`s in tandom.
+
+```js
+import { Collapser } from '@newrelic/gatsby-theme-newrelic'`
+```
+
+**Props**
+
+| Prop          | Type                    | Required | Default | Description                                                                                                                  |
+| ------------- | ----------------------- | -------- | ------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `children`    | enum                    | yes      |         | The content that will be hidden or revealed when the user interacts with the `Collapser`.                                    |
```

Comment:
Shouldn't `children` be of the `node` type? If not, we should update the propTypes in the component.

Reply:
Oops, good catch. I copy/pasted this from another component and forgot to update this one.

Reply:
Fixed in 94a8519